trackintel/analysis/speed.py

```diff
@@ -0,0 +1,83 @@
+import trackintel as ti
+import numpy as np
+import pandas as pd
+
+
+def speed_positionfixes(positionfixes):
+    """
+    Compute speed per positionfix
+
+    Parameters
+    ----------
+    positionfixes : GeoDataFrame (as trackintel positionfixes)
+        The positionfixes have to follow the standard definition for positionfixes DataFrames.
+
+    Returns
+    -------
+    pfs: GeoDataFrame (as trackintel positionfixes)
+        The original positionfixes with a new column ``[`speed`]``. The speed is given in km/h
+
+    Notes
+    -----
+    The speed at one positionfix is computed from the distance and time since the previous positionfix. For the first
+    positionfix, the speed is set to the same value as for the second one.
+    """
+    pfs = positionfixes.copy()
+    crs_is_projected = ti.geogr.distances.check_gdf_planar(pfs)
```

Comment:
I would call this `if_planer_crs`, as it is called in `_predict_transport_mode_simple_coarse()` in labelling.
`crs_is_projected` might cause confusion in "is projected" or "is in the projected coordinate system".

Reply:
Done